esx_service/cli/vmdkops_admin.py

```diff
@@ -420,8 +420,8 @@ def commands():
                     'help': 'Volume to set options for, specified as "volume@datastore".',
                     'required': True
                 },
-                '--tenant': {
-                    'help': 'Name of the tenant the volume belongs to.',
+                '--vm-group': {
```

Comment:
OK, @lipingxue - please fix the test and it will be good to go (CI needs to pass, ot the failures have to be known ones & preexisting)